tests/foreman/ui/test_reporttemplates.py

```diff
@@ -345,3 +403,60 @@ def test_negative_nonauthor_of_report_cant_download_it(session):
     :expectedresults: Report can't be downloaded. Error.
     :CaseImportance: High
     """
+
+
+@tier3
+def test_positive_generate_entitlements_reports_multiple_formats(session, content_setup):
+    """Generate reports using the Entitlements template in html, yaml, json, and csv format.
+
+        :id: b268663d-c213-4e59-8f81-61bec0838b1e
+
+
+        :setup: Installed Satellite with Organization, Activation key,
+                Content View, Content Host, and Subscriptions.
+
+        :steps:
+            1. Monitor -> Report Templates
+            2. Entitlements -> Generate
+            3. Click the dropdown to select output format
+            4. Submit
+
+        :expectedresults: reports are generated containing all the expected information
+                          regarding Entitlements for each output format.
+
+        :CaseImportance: High
+    """
+    with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+        vm.install_katello_ca()
+        vm.register_contenthost(content_setup[0].label, content_setup[1].name)
```

Comment:
This will be easier to read:
```suggestion
        org, ak = content_setup
        vm.register_contenthost(org.label, ak.name)
```